client/index.jsx

```diff
@@ -19,17 +22,31 @@ class App extends React.Component {
     $.get({
       url: `/description?prod_id=${prod}`,
       success: (res) => {
-        console.log(res)
         this.setState({
           data:res
         })
       },
       dataType: 'JSON'
     })
   }
-
+//conditional rendering here is a work around for an asycn bug that was crashing the app.  The first render had nothing in state so was passing undefined.  Now code waits for get request res before rendering components
   render() {
-    return <div>Hello React,Webpack 4 & Babel 7!</div>
+    if (this.state.data.length < 1) {
+      return <p>loading</p>
+    }
```

Comment:
Interesting solution to this issue. My solution was to put default values into my default state to allow the first render to still occur before the first GET request.